core/src/test/java/org/apache/brooklyn/core/test/BrooklynAppLiveTestSupport.java

```diff
@@ -36,10 +36,17 @@
     @Override
     @BeforeMethod(alwaysRun=true)
     public void setUp() throws Exception {
+
         if (mgmt!=null) {
             app = mgmt.getEntityManager().createEntity(newAppSpec());
         } else {
-            mgmt = new LocalManagementContextForTests(BrooklynProperties.Factory.newDefault());
+
+            LocalManagementContextForTests.Builder builder = LocalManagementContextForTests.builder(true).
+                    setOsgiEnablementAndReuse(true, true);
+            if (true) {
+                builder.useDefaultProperties();
+            }
```

Comment:
Does not look like `if` statement needed.

Reply:
I mean, `if (true)`

Reply:
I'm not sure where this change has come from - I don't have it locally?

Reply:
I must have messed up a rebase